### PR TITLE
Add wifi/bluebooth info & add a type.scss for PNI

### LINKED ISSUE #5249: Add “Wi-Fi” and "Bluetooth" icons where applicable
@jamilasnell commented on [Wed Sep 16 2020](https://github.com/mozilla/network-pulse/issues/1417)

### Add “Wi-Fi” and "Bluetooth" icons where applicable
- [ ] If answer is 'Yes' then show the icon (if the answer is no then hide the icon) - [icons](https://drive.google.com/drive/folders/1kPMOO9YoMybLN-MqziLUsmGE2EQ3hAVp?usp=sharing)
- [ ] Icon and text should be Gray 60 ($gray-60) #666666

### Desktop 
image

### Mobile
image

---

**🗒  Design -> Dev Handoff Checklist**

- [x] Design has been finalized
- [x] Link to original design ticket #5152
- [X] Link to related ticket #5234
- [x] Link to [Sketch file](https://share.goabstract.com/584e7002-4d07-4ce5-aafe-dc825e076a27?sha=f0b8ca1b588425d1ef83c57a9f4126c8ff97648e)
- [x] Design changes been merged into the master Sketch file
- [x] Desktop and mobile mockups are included
- [x] Assets are included - [icons](https://drive.google.com/drive/folders/1kPMOO9YoMybLN-MqziLUsmGE2EQ3hAVp?usp=sharing)
- [x] Hover state for elements are designed - n/a
- [x] Other states for elements are designed (e.g., error state for form) - n/a

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #5249 

- wifi + bluebooth: https://foundation-s-issue-5249-xkoaai.herokuapp.com/en/privacynotincluded/products/character-lot/
- wifi only: https://foundation-s-issue-5249-xkoaai.herokuapp.com/en/privacynotincluded/products/employee-entire/
- bluetooth only:  https://foundation-s-issue-5249-xkoaai.herokuapp.com/en/privacynotincluded/products/direction-white/
- none: https://foundation-s-issue-5249-xkoaai.herokuapp.com/en/privacynotincluded/products/heart-old/

### CODE REVIEW

File: network-api/networkapi/buyersguide/templates/product_page.html [hunk 1]
```diff
@@ -36,8 +36,17 @@
     <div class="product-detail bg-white py-3 py-md-4 px-md-5 {{section_class}}">
       <div class="row intro mb-4">
         <h1 class="h1-heading col-12">{{product.name}}</h1>
-        <a id="product-company-url" class="product-price small-link col-md-8 mb-1 mb-md-0" href="{{product.url}}" target="_blank">{{product.company}} {% if product.product_type == "general" %}${% endif %}{{product.price}}</a>
-        <div class="col-md-4 text-md-right">TODO: wifi and bluetooth</div>
+        <div class="col-12 d-flex flex-column flex-md-row justify-content-between">
+          <a id="product-company-url" class="product-price pni-product-intro-large mb-2 mb-md-0" href="{{product.url}}" target="_blank">{{product.company}} {% if product.product_type == "general" %}${% endif %}{{product.price}}</a>
+          <div>
+            {% if product.uses_wifi %}
+              <div class="connectivity-requirement use-wifi pni-product-intro-large">{% trans "WiFi" %}</div>
```
Comment (Pomax) [inline network-api/networkapi/buyersguide/templates/product_page.html:43]: Can we make them `<span>` or `<p>` instead, since they are text elements?

**Review by Pomax — CHANGES_REQUESTED** [review]

**Comment by kristinashu** [comment]
Nice, the icons are appearing as expected in all the different possible states. But for design could you please:

- [ ] Add the `-` between `Wi-Fi` (according to Theo this is more common in other languages)
- [ ] Vertically center the icons and the text (in the PR it looks like they are aligned at the bottom but in the design their centers align)  

image

**Review by kristinashu — CHANGES_REQUESTED** [review]
See comment.

**Comment by mmmavis** [comment]
PR updated!

**Review by kristinashu — APPROVED** [review]
Nice!

**Review by Pomax — APPROVED** [review]

**Pull request merged**